### PR TITLE
Optimization/homepage

### PR DESCRIPTION
**Pull request by jomarmontuya**

### CODE REVIEW

File: src/blocks/Image/ZestyImage.js [hunk 1]
```diff
@@ -25,8 +24,15 @@ const ZestyImage = ({
...
   attributes = {},
 }) => {
-  /* Taking the options object and converting it into a query string. */
+  //  check if exist and assign values to options object
+  if (width) {
+    options.width = width;
+  }
+  if (height) {
+    options.height = height;
+  }
```
Comment (jomarmontuya) [inline src/blocks/Image/ZestyImage.js:34]: assign the width and height to the options object if exist, in this way I can only pass  width and height from the parent component and will return consistent height and width from CDN and `````` tag

Comment (jomarmontuya) [reply src/blocks/Image/ZestyImage.js:34]: image


example call from parent

Comment (jomarmontuya) [reply src/blocks/Image/ZestyImage.js:34]: HTML rendered 

image

Comment (jomarmontuya) [reply src/blocks/Image/ZestyImage.js:34]: Sample call with additional options

image

HTML rendered with additional options

image

**Review by ardeay — APPROVED** [review]

**Comment by ardeay** [comment]
good

**Pull request merged**